decko-rails/lib/decko/engine.rb

```diff
@@ -46,9 +46,16 @@ class Decko::Engine < Rails::Engine
       ActiveSupport.on_load(:active_record) do
         pths = Wagn.paths['request_log'] and Decko::Engine.paths['request_log'] = pths
         pths = Wagn.paths['log'] and Decko::Engine.paths['log'] = pths
-        Cardio.add_card_paths Decko::Engine.paths, Rails.application.config, Wagn.root
+        Cardio.card_config Rails.application.config, Decko::Engine.paths, Wagn.root, Rails.cache
         ActiveRecord::Base.establish_connection(Rails.env)
       end
+      ActiveSupport.on_load(:after_initialize) do
+        begin
+          require_dependency 'card' unless defined?(Card)
+        rescue ActiveRecord::StatementInvalid => e
+          Rails.logger.warn "database not available[#{Rails.env}] #{e}"
+        end
```

Comment:
I wish there were an easy way to see the updated diffs (now that stuff is
merged in). sigh.

at least we're in the hundreds now (diff lines) and not tens of thousands ;)

On Thu, Feb 12, 2015 at 5:57 PM, Gerry Gleason <notifications@github.com>
wrote:

> In decko-rails/lib/decko/engine.rb
> <https://github.com/wagn/wagn/pull/311#discussion_r24636438>:
>
> >          ActiveRecord::Base.establish_connection(Rails.env)
> >        end
> > +      ActiveSupport.on_load(:after_initialize) do
> > +        begin
> > +          require_dependency 'card' unless defined?(Card)
> > +        rescue ActiveRecord::StatementInvalid => e
> > +          Rails.logger.warn "database not available[#{Rails.env}] #{e}"
> > +        end
>
> I verified this (wagn console works on 1_15 plus this addition. Let me now
> what you want to do. You could just add it to 1_15, or I can make a branch
> for this and maybe a few more critical details.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wagn/wagn/pull/311/files#r24636438>.
>



-- 
Ethan McCutchen
One of the Wagneers, Wagn.org

Wagn. How pioneers roll.

s: ethan.mccutchen
t: @intogreater